test/integration/functional_test.go

```diff
@@ -2055,6 +2056,41 @@ users:
 	}
 }
 
+// validateConfigUpdateCmd asserts basic "config update" command functionality
+func validateConfigUpdateCmd(ctx context.Context, t *testing.T, profile string) {
+	newMinikubeHomeDir := t.TempDir()
+	newMinikubePath := filepath.Join(newMinikubeHomeDir, ".minikube")
+
+	if err := cp.Copy(localpath.MiniPath(), newMinikubePath); err != nil {
+		t.Fatalf("failed to copy %s: %v", localpath.MiniPath(), err)
+	}
+
+	c := exec.CommandContext(ctx, Target(), "-p", profile, "config", "update")
+	c.Env = append(os.Environ(), fmt.Sprintf("MINIKUBE_HOME=%s", newMinikubeHomeDir))
+
+	// Run `minikube config update`
+	rr, err := Run(t, c)
+	if err != nil {
+		t.Errorf("failed to run minikube config update: args %q: %v", rr.Command(), err)
+	}
+
+	// Make sure the machine config paths have been correctly updated by checking the command output
+	if !bytes.Contains(rr.Stdout.Bytes(), []byte("Machine config paths has been updated")) {
```

Comment:
```suggestion
	if !strings.Contains(rr.Stdout.String(), "Machine config paths has been updated") {
```
I think this would make more sense as a string compare

Reply:
Done.